index.bs

```diff
@@ -0,0 +1,40 @@
+<pre class='metadata'>
+Title: The TODO API
+Shortname: todo
+Level: None
+Status: w3c/UD
+Repository: explainers-by-googlers/todo-your-repo-name
+URL: https://explainers-by-googlers.github.io/todo-your-repo-name
+Editor: TODO: Your Name, Google https://google.com, TODO@google.com
+Abstract: TODO: A short description of your spec, one or two sentences.
+Markup Shorthands: markdown yes, css no
+Complain About: accidental-2119 yes, missing-example-ids yes
+Assume Explicit For: yes
+Die On: warning
+WPT Path Prefix: TODO-API-LABEL
+WPT Display: closed
+Include MDN Panels: if possible
+Include Can I Use Panels: yes
+</pre>
+
+Introduction {#intro}
+=====================
+
+For now, see the [explainer]([REPOSITORYURL]).
+
+See [https://garykac.github.io/procspec/](https://garykac.github.io/procspec/),
+[https://dlaliberte.github.io/bikeshed-intro/index.html](https://dlaliberte.github.io/bikeshed-intro/index.html),
+and [https://speced.github.io/bikeshed/](https://speced.github.io/bikeshed/) to get started on your
+specificaton.
+
+<wpt rest></wpt>
```

Comment:
Hmmm, @tabatkins this spelling produces an ["Attribute “rest” not allowed on element “details” at this point." error in spec-prod's markup validation](https://github.com/explainers-by-googlers/template/actions/runs/7023867643/job/19111358245?pr=1). Using `<wpt-rest>` as suggested in https://speced.github.io/bikeshed/#elementdef-wpt-rest fails Bikeshed with
```
FATAL ERROR: Couldn't find any tests with the path prefix 'TODO-API-LABEL'.
```

I could just remove this or comment this out, since the issue tells people what to do once they set their WPT Path Prefix. What do you think?

Reply:
Removed. :)